optaplanner-docs/src/main/docbook/en-US/Chapter-Quick_start/Chapter-Quick_start.xml

```diff
@@ -256,12 +308,12 @@
           our business requirements. Further down, we 'll look into 2 alternatives to calculate the score: using a
           simple Java implementation or using Drools DRL.</para>
 
-          <programlisting language="xml">  &lt;scoreDirectorFactory&gt;
-    &lt;scoreDefinitionType&gt;HARD_SOFT&lt;/scoreDefinitionType&gt;
-    &lt;easyScoreCalculatorClass&gt;org.optaplanner.examples.cloudbalancing.solver.score.CloudBalancingEasyScoreCalculator&lt;/easyScoreCalculatorClass&gt;
-    &lt;!--&lt;scoreDrl&gt;org/optaplanner/examples/cloudbalancing/solver/cloudBalancingScoreRules.drl&lt;/scoreDrl&gt;--&gt;
-    &lt;initializingScoreTrend&gt;ONLY_DOWN&lt;/initializingScoreTrend&gt;
-  &lt;/scoreDirectorFactory&gt;</programlisting>
+          <programlisting language="xml"> <![CDATA[<scoreDirectorFactory>
```

Comment:
in "...language="xml"> <![CDATA[<sco...", let's put the identation space inside the CDATA. It should be the same amount of spaces for <scoreDirectorFactory> as for the closing tag </scoreDirectorFactory>, I 'll fix that (could also be that I broke that with the tab replacement, doesn't matter).